### PR TITLE
[CI:DOCS] docs: include Go bindings tutorial

### PR DESCRIPTION
**Pull request by lsm5**
Include the Go bindings blog post as a tutorial

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@TomSweeneyRedHat

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by lsm5** [comment]
> Probably ought to add a link here too: https://github.com/containers/podman/blob/master/docs/tutorials/README.md

done, PTAL.

**Comment by TomSweeneyRedHat** [comment]
LGTM
thanks @lsm5 and @ParkerVR for putting this together.

**Comment by edsantiago** [comment]
@lsm5 the trick to avoiding CI is to put `[CI:DOCS]` (brackets are customary but not required) in the subject of your PR. Just an FYI for next time. ( /me sneaks back out - I'm not qualified to review the content )

File: docs/source/Tutorials.rst [hunk 1]
```diff
@@ -10,3 +10,4 @@ Here are a number of useful tutorials to get you up and running with Podman. If
 * `How to sign and distribute container images using Podman <https://github.com/containers/podman/blob/master/docs/tutorials/image_signing.md>`_: Learn how to setup and use image signing with Podman.
 * `Podman remote-client tutorial <https://github.com/containers/podman/blob/master/docs/tutorials/remote_client.md>`_: A brief how-to on using the Podman remote-client.
 * `How to use libpod for custom/derivative projects <https://github.com/containers/podman/blob/master/docs/tutorials/podman-derivative-api.md>`_: How the libpod API can be used within your own project.
+* `How to use Podman's Go bindings <https://github.com/containers/podman/blob/master/docs/tutorials/podman-go-bindings.md>`: A brief how-to on using Podman's Go bindings in external applications.
```
Comment (edsantiago) [inline docs/source/Tutorials.rst:13]: Okay one more comment: the underscore is, IIRC, a necessary part of an RST link.

File: docs/tutorials/podman-go-bindings.md [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+Open another terminal window and check if the Podman socket exists:
+
+```bash
+$ ls /run/user/${UID}/podman
+podman.sock
+```
+
+If you’re running the system service as root, podman.sock will be found in /run/podman:
+```bash
+$ ls /run/podman
```
Comment (edsantiago) [inline docs/tutorials/podman-go-bindings.md:117]: Suggestion (only if you need to make changes elsewhere): use `#` for a root prompt. Makes it very easy on the reader.

File: docs/source/Tutorials.rst (hunk 1 above)
Comment (lsm5) [reply docs/source/Tutorials.rst]: done.

File: docs/tutorials/podman-go-bindings.md (hunk 2 above)
Comment (lsm5) [reply docs/tutorials/podman-go-bindings.md]: done.

**Comment by lsm5** [comment]
> @lsm5 the trick to avoiding CI is to put `[CI:DOCS]` (brackets are customary but not required) in the subject of your PR. Just an FYI for next time. ( /me sneaks back out - I'm not qualified to review the content )

done.

**Pull request merged**